zk.el

```diff
@@ -778,13 +778,18 @@ Optionally call a custom function by setting the variable
 
 ;;; Insert Link
 
-(defun zk--format (format id title)
+(defun zk--format-default (format id title)
```

Comment:
I don't like `default` in the name, because it's not descriptive.

What about `zk-format-link-and-title`, since that's what it does?

Reply:
Yes, it does not seem right. My concern is that `zk-format-link-and-title` is misleading, since it does not necessarily format just links (i.e. `[[id]]`). How about `zk-format-id-and-title`?

Should `zk--format` be use-facing as well, then?

Reply:
`zk-format-id-and-title` seems fine, and I don't see a reason to make `zk--format` public.